docs/howto/create-a-hollow-jar/index.adoc

```diff
@@ -1,27 +1,28 @@
 [#creating-a-hollow-jar]
 = Creating a hollow JAR
 
-The xref:hollow-jar[hollow JAR] is one method of packaging an application for
-execution with WildFly Swarm.
+The xref:hollow-jar[hollow JAR] is one method of packaging an application for execution with WildFly Swarm.
 
 .Prerequisites
 
 * A Maven-based application with a `pom.xml` file.
 
 .Procedure
 
-. Add the `wildfly-swarm-maven-plugin` to your `pom.xml` in a `<plugin>` block,
-  with an `<execution>` specifying the `package` goal.
-  In addition to that, put `true` in the `hollow` property of the
-  `<configuration>` element:
+. Add the `wildfly-swarm-maven-plugin` to your `pom.xml` in a `<plugin>` block, with an `<execution>` specifying the `package` goal.
```

Comment:
I know this isn't changed here, but the `artifactId` of the plugin is `wildfly-swarm-plugin`, not `wildfly-swarm-maven-plugin`. So either this should be "Add the `wildfly-swarm-plugin` to your `pom.xml`", or "Add the WildFly Swarm Maven plugin to your `pom.xml`".

Reply:
Oh, my bad, will fix. Thanks.

Reply:
It's not your bad (at least not in this PR :-D, not sure who wrote it), but thanks.

Reply:
+1 on `wildfly-swarm-plugin` change.